packages/ra-ui-materialui/src/list/filter/FilterListWrapper.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { ReactNode } from 'react';
+import { Box, BoxProps, Typography } from '@mui/material';
+import { useTranslate } from 'ra-core';
+
+export const FilterListWrapper = (props: FilterListWrapperProps) => {
```

Comment:
What's the purpose of extracting this component?

Reply:
I found it neat to separate the logic from the presentaiton. `<AutoSubmitFilterForm>` is a logic-only component, which makes it more versatile as it can be used in any UI.
But I still wanted an easy way to use `<AutoSubmitFilterForm>` alongside `<FilterList>` (which was the objective in the first place), so reusing part of the UI from `<FilterList>` felt like a good way to achieve that while keeping the DX not too bad.